### PR TITLE
Add tests to evaluate classification accuracy

### PR DESCRIPTION
**Pull request by yanirs**
Add tests for `LangdetectService` that evaluate its accuracy on various text lengths and types. The tested parameters include:
* Text length, with emphasis on short texts in the 5-20 characters range (simulating search queries)
* Text type, represented by two datasets that cover all the languages supported by the plugin: translations of the Universal Declaration of Human Rights and of the WordPress interface
* Language profile: default or short
* Supported languages: default or all

The Python code used to generate the datasets is also included. See `scripts/README.md` for running instructions.

To view the output of the new tests, run `gradle test --info`.

### CODE REVIEW

File: src/test/java/org/xbib/elasticsearch/index/mapper/langdetect/DetectLanguageTest.java [hunk 1]
```diff
@@ -34,14 +55,271 @@ public void testKorean() throws IOException {
         testLanguage("korean.txt", "ko");
     }
 
+    @Test
+    public void testUdhrAccuraciesDefaultProfileDefaultLanguages() throws IOException {
```
Comment (yanirs) [inline src/test/java/org/xbib/elasticsearch/index/mapper/langdetect/DetectLanguageTest.java:59]: These tests are a bit repetitive, but I'm not sure whether looping over these parameters in a loop would be much cleaner/clearer.

File: scripts/run.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+    :param out_path: output path for the dataset file, which will be written in tab-separated format with two
+                     columns: language code and text.
+    :param texts_per_language: number of texts to retain per langauge. The output file will contain up to this number of
+                               texts per language, excluding URL translations and word lists. The longest texts for
+                               each language are retained.
+    """
+    url_template = 'https://translate.wordpress.org/projects/wp/4.6.x/{}/default/export-translations?format=json'
+    requests_session = requests.Session()
+    wp_placeholder_pattern = re.compile(r'(%\d*\$?[sd])|(###[A-Z_]+###)')
```
Comment (yanirs) [inline scripts/run.py:144]: This doesn't completely filter all placeholders, but I think that it's not necessarily a bad thing to leave some Latin characters in the texts, as they also occur in organically-created texts.

Comment (stambizzle) [reply scripts/run.py:144]: 👍

File: src/test/java/org/xbib/elasticsearch/index/mapper/langdetect/DetectLanguageTest.java (hunk 1 above)
Comment (stambizzle) [reply src/test/java/org/xbib/elasticsearch/index/mapper/langdetect/DetectLanguageTest.java:59]: Seems like a toss up to me. I think this implementation is fine.

**Review by stambizzle — APPROVED** [review]

**Pull request merged**